### DISCUSSION TITLE
OpenPGP Signature

### DISCUSSION THREAD

**Discussion by farfallosa** [protz/thunderbird-conversations#2095] [Q&A]
Hello,
using you awesome APP I only miss any information towards OpenPGP signature or encryption. I do have to open mails in classic mode/ view to get this information. Is there a way to see this like DKIM Information etc.?

Thanks in advance and regards, Will

**Comment by stratosjl** [comment]
The worst part is that when you use PGP, it does not show any attachments. This is a bug that's known for years, but it has low visibility for the author. I have to deactivate and reactivate this plugin every time i want to see a signed or encrypted message.

**Comment by admick054-maker** [comment]
Yes, this is a huge problem, and it's very annoying. These messages aren't displayed (the content is missing). The problem is becoming more significant as we receive more and more encrypted messages in our company, and clicking the "View in classic form" option every time is quite irritating. Is there a chance this will be fixed in a future plugin update?

**Reply by Standard8** [reply]
There's a chance. However it is likely quite a bit of work to get it right and I don't use encryption myself. Thunderbird's possibly adding a conversation view in future which would presumably handle this (though I suspect that's a couple more years away).

Given this is free, and it sounds like your company wants it, maybe your company might like to employ someone to write the code?
